source/agora/consensus/EnrollmentManager.d

```diff
@@ -1024,3 +1028,67 @@ unittest
     assert(result_image.hash == man.cycle.preimages[100]);
     assert(result_image.distance == 1100);
 }
+
+/// Test `PreImageCycle` consistency between seeds and preimages
+unittest
+{
+    // Note: This was copied from `EnrollmentManager` constructor and should
+    // be kept in sync with it
+    auto cycle = EnrollmentManager.PreImageCycle(
+        /* nounce: */ 0,
+        /* index:  */ 0,
+        /* seeds:  */ PreImageCache(EnrollmentManager.NumberOfCycles, Enrollment.ValidatorCycle),
+        /* preimages: */ PreImageCache(Enrollment.ValidatorCycle, 1)
+    );
+
+    auto secret = Scalar.random();
+    Scalar fake_secret; // Used whenever `secret` *shouldn't* be used
+    foreach (uint cycleCount; 0 .. 10)
```

Comment:
The `cycle` in the `cycleCount` is confusing with the `Cycle` in the `NumberOfCycles`. I'd like you to change `cycleCount` into another word like `cycleGroupCount`

Reply:
Done